core/src/commonMain/kotlin/io/github/aag/core/domain/OperationResult.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.aag.core.domain
+
+
+sealed class OperationResult<out S : Any?, out E : Any?> {
```

Comment:
Maybe It'll be better to create a wrapper in this style - https://github.com/skydoves/sandwich/blob/fc6d72436da3a2ca23ed74466fd922c80f616521/sandwich/src/main/kotlin/com/skydoves/sandwich/ApiResponse.kt ?
So we can use it specifiyng only one generic parameter instead of two in current implementation